src/cloud-api-adaptor/docs/consuming-prebuilt-podvm-images.md

```diff
@@ -2,22 +2,20 @@
 
 The project has automation in place to build container images that contain the qcow2 file for the various Linux distributions and cloud providers supported. Those images are re-built for each release of this project.
 
-You can find the images available at https://quay.io/organization/confidential-containers with the *podvm-[generic|vsphere]-[DISTRO]-[ARCH]* name pattern.
+>Note: The published qcow2 images doesn't have any TEE support. For specific TEE support, you'll need to build your own pod VM image or used the published Azure, AWS and GCP images.
+Refer to the coco [website](https://confidentialcontainers.org/docs/examples/) for more details.
+
+You can find the images available at https://quay.io/organization/confidential-containers with the *podvm-generic-[DISTRO]-[ARCH]* name pattern.
 
 For example:
-- https://quay.io/repository/confidential-containers/podvm-generic-ubuntu-amd64 hosts the Ubuntu images that can be used with all providers (except vsphere).
-- https://quay.io/repository/confidential-containers/podvm-vsphere-ubuntu-amd64 hosts the Ubuntu images that can be used with vsphere provider.
+- https://quay.io/repository/confidential-containers/podvm-generic-ubuntu-amd64 hosts the Ubuntu images that can be used with all providers.
 - https://quay.io/repository/confidential-containers/podvm-generic-ubuntu-s390x hosts the Ubuntu images that can be used for s390x architecture.
-- https://quay.io/repository/confidential-containers/podvm-generic-centos-amd64 hosts the CentOS images that can be used with all providers (except vsphere).
-
->Note: images for CentOS Stream haven't been built since the release 0.8.0 of this project.
 
 The easiest way to extract the qcow2 file from the podvm container image is using the [`download-image.sh`](../podvm/hack/download-image.sh) script. For example, to extract the file from the *podvm-generic-ubuntu-amd64* image:
```

Comment:
I don't know if this is worth noting that this only works for the packer based podvm images (so distro == ubuntu) - the mkosi ones are pulled with oras?

Reply:
The instructions for mkosi are basically:
- Install oras: https://oras.land/docs/installation/
- Pull the image with: `oras pull <podvm image e.g. quay.io/confidential-containers/podvm-generic-fedora-amd64:v0.13.0>`
- Untar what you've just pulled: `tar xvJpf podvm.tar.xz`

At that point you'll have the qcow in your directory.

Reply:
Added the details..ptal